examples/provider/bid/audience-network/src/audience-network-bid-delegate.js

```diff
@@ -0,0 +1,138 @@
+/**
+ * @file Audience Network BidDelegate.
+ */
+import 'whatwg-fetch';
+
+const name = 'audience-network';
+
+/**
+ * Does this slot contain valid parameters?
+ * @param {Object} slot
+ * @returns {Boolean}
+ */
+const validateSlot = slot =>
+  typeof slot === 'object' &&
+  typeof slot.name === 'string' &&
+  typeof slot.elementId === 'string' &&
+  Array.isArray(slot.sizes) &&
+  slot.sizes.length > 0 &&
+  typeof slot.getParam === 'function' &&
```

Comment:
a `Slot` inherits `getParam()` from `PubfoodObject` (_http://pubfood.org/api-reference#pubfood-model-Slot-getParam_)

`slot.getParam` here will be undefined.

`Object.assign` below will copy only enumerable _**owned**_ properties: being inherited, `getParam` will not be copied with `Object.assign`.

`Object.create` would do something more like you might want i.e. this will copy the prototype.